### DISCUSSION TITLE
Which provider do you use to run your VPS with AzuraCast?

### DISCUSSION THREAD

**Discussion by Heimatradio** [AzuraCast/AzuraCast#7462] [General]
Which provider do you use to run your VPS?
Which provider do you not have problems with stolen RAM memory?
please write in comments your provider, config (how much vcores, ram, etc.)
This might help me find a good vps.
Thank you in advance for your effort

**Comment by LordHelmchen666** [comment]
I guess you are also located in germany?
I use strato.de VC6 (6 Cores/16GB Ram/500GB SSD) 11€/month without problems. Two Stations with Autocue and masterme enabled. 
![Screenshot_20240923_181355](https://github.com/user-attachments/assets/a6c5d4f8-a241-42ba-a129-c8ea4d3781da)

**Reply by Heimatradio** [reply]
I would not have thought of Strato (Hätte ich nicht gedacht von Strato, Danke)

![contabo](https://github.com/user-attachments/assets/32e1588e-38f3-46d5-98db-8a423d27b54f)
I have one from Contabo, among others, the stolen CPU resources are causing problems there
running only 4 Streams, 2 with StereoTool

**Reply by LordHelmchen666** [reply]
Contabo is known for "overloading" their VPS systems. If you don't need much HDD space ovh.de is also good performing. I also had a short time a VPS from servdiscount.com. It was also not bad.

**Comment by Heimatradio** [comment]
this is an other one running at GenX with 8 Stations, 4 with StereoTool

![genx](https://github.com/user-attachments/assets/4aaa2097-18b9-431b-a45c-80c6b2e3412f)

Very strange how the servers behave differently

**Reply by LordHelmchen666** [reply]
Could you share which CPU they use? On strato.de i have "AMD EPYC-Milan"

**Reply by Heimatradio** [reply]
at GenX i see only AuthenticAMD, the vserver is part of a complete radio package at Genx.de
and it runs surprisingly well

**Reply by Heimatradio** [reply]
The server at GenX is not mine, but i'm admin there and i have temporaly moved the streams to it because
the problems at contabo

I think I'll test the one at Strato, the VC6-12 now has 12GB RAM, but that makes no difference to 16GB.

**Comment by Heimatradio** [comment]
![strato](https://github.com/user-attachments/assets/e0a8250e-2f3b-4a81-a728-7257e0f4f38a)
nice nice nice @LordHelmchen666  , this is the new VC6-12 at Strato with 360GB SSD
i have installed my own cloudpanel from cloudpanel.io , now 6 shoutcast streams (3 with StereoTool) on AzuraCast, also one little
wordpress-site is running on the server.

**Reply by LordHelmchen666** [reply]
Looks very promising.
